CronosPlaySdk/Plugins/CronosPlayUnreal/Source/CronosPlayUnreal/Private/PlayCppSdkBPLibrary.cpp

```diff
@@ -177,6 +182,149 @@ UTexture2D *UPlayCppSdkBPLibrary::GenerateQrCode(FString string) {
   return texture;
 }
 
+// ipfs:// -> https://ipfs.io/ipfs/
+FString convert_ipfs_url(const FString &src) {
+  FString dst = src.Replace(TEXT("ipfs://"), TEXT("https://ipfs.io/ipfs/"));
```

Comment:
maybe the http gateway could be configurable instead of hardcoded to the public one?

Reply:
ok